semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -203,12 +203,13 @@ def to_creation_time_attributes(
         sampling_relevant_attrs: typing.List[SemanticAttribute],
         output: io.StringIO,
     ):
-        """Renders list of attributes that MUST be provided at creation time
+        """Renders list of attributes that SHOULD be provided at creation time
         :return:
         """
         if sampling_relevant_attrs:
             output.write(
-                "\nFollowing attributes MUST be provided **at span creation time** (when provided at all):\n\n"
+                "\nFollowing attributes SHOULD be provided **at span creation time** (when provided at all), "
+                + "so they can be considered for sampling decisions:\n\n"
```

Comment:
Please also bump the version so we can release it right away to be referenced on https://github.com/open-telemetry/opentelemetry-specification/pull/1916.
You could run the complete wording by the @open-telemetry/specs-approvers over there again to double-check but I assume it should be fine now.